docs/output.md

```diff
@@ -243,6 +222,7 @@ The output from this step will be placed in the output folder root.
   - Reports generated by the pipeline: `pipeline_report.html`, `pipeline_report.txt` and `software_versions.yml`. The `pipeline_report*` files will only be present if the `--email` / `--email_on_fail` parameter's are used when running the pipeline.
   - Metadata file with software versions, environment information and pipeline configuration for debugging: `metadata.json`
```

Comment:
If `PIXELATOR_COLLECT_METADATA` was removed, is this still valid?

Reply:
Good catch! This should be removed too :+1: